\chapter{\label{ch:3}Application of Spatial Transcriptomic Technologies in Breast Cancer}

\minitoc

\section{Introduction}


\section{Results}
%\label{app:imaging}


\subsection{Comparison of Emerging Spatial Transcriptomic Technologies}
Slide-seq
HDST
ReadCoor/FISSEQ
CARTANA
NanoString

spatial resolution vs cost vs availability



\subsection{Application of Slide-seq in Breast Cancer}
\subsubsection{Development of Slide-seq Pipeline in Breast Cancer Biopsy Samples}

After careful consideration and review of all spatial transcriptomic technologies, I elected to proceed with the use of Slide-seq on treatment naive fresh frozen breast cancer biopsies. Slide-seq is a novel technology which is not routinely available within Europe. The technology cannot be purchased from a commercial provider. At present, it is only available within the Macosko or Regev lab at the Broad Institute of MIT and Harvard, Boston, USA.


Therefore, I independently established a collaboration directly with the Macosko lab in order to proceed with this research program for my DPhil. I also established a bespoke training program within the Macosko lab for which I travelled to Boston, USA in November 2019. The training objectives were:

a. To develop expertise in the production of spatially barcoded beads and the effective uniform placement of beads onto glass slides. \\
b. To understand the process by which each bead is spatially profiled using in situ sequencing technology. \\
c. To develop best practice techniques in sample selection, tissue placement onto spatial slides and the downstream processing for library generation.

Upon completion of the training program in the US, I returned back to Oxford with a collection of pilot spatial slides to set up the novel experimental pipeline with the associated specialised reagents. The establishment of the in situ sequencing platform for the spatial slides required several years of optimisation by the Macosko lab. It would require \textasciitilde 1-2 years to develop an equivalent optimised platform in Oxford. My priority for the DPhil was to explore the application of spatial transcriptomic technologies in clinical breast cancer samples. Therefore, I aimed to establish a pipeline in which the fully profiled spatial slides would be prepared by the Macosko lab and I would process the spatial slides on clinical breast cancer samples in Oxford.

Each spatial slide, referred to as a 'puck', is produced from a 0.17 mm thick glass coverslip which is fragile. I developed expertise in the correct and gentle handling of the coverslips to avoid the creation of cracks during the protocol. Cracks created during the tissue placement stage, in particular, would introduce spatial artefacts. Consequently, the correct selection of tweezer for handling of the pucks was essential. An iterative selection process using multiple pilot slides allowed the identification of a fine stainless steel tweezer (\#T4537-1EA Sigma) as optimal for puck handling.

Slide-seq is suitable only for fresh frozen tissue. The sectioning of fresh frozen tissue is performed within a cryostat. Standard practice entails embedding tissue within OCT to provide a stable base from which sectioning is performed. However, OCT interferes with RNA hybridisation onto the puck. The placement of tissue within a cryostat was accordingly modified as part of the Slide-seq protocol. A small volume of OCT is placed to allow for tissue stabilisation. However, the upper layer of the tissue specimen is kept free of OCT. 

Breast cancer clinical biopsies are \textasciitilde 4 mm diameter and of variable shape and adipose composition. The physical characteristics of the biopsies presented significant additional challenges for accurate tissue placement onto the puck. Tissue sectioning and placement was performed with Dr. Esther Bridges, a very senior and experienced scientist. The skill and technical acumen of Dr. Bridges was an essential component required for the successful establishment of the Slide-seq pipeline.


\begin{figure}[!htb]
	\minipage{0.32\textwidth}
		\left
		\includegraphics[width=0.7\textwidth, angle=270]{figures/slideseq_cryostat_1.jpg} \hfill
		\caption[Biopsy orientation in cryostat]{Biopsy \\ orientation in cryostat}
		\label{fig:slideseq_cryostat_1}
	\endminipage\hfill
	\minipage{0.32\textwidth}
		\centering
		\includegraphics[width=0.7\textwidth, angle=270]{figures/slideseq_cryostat_2.jpg} \hfill
		\caption[Sample 1]{Sample 1}
		\label{fig:slideseq_cryostat_2}
	\endminipage\hfill
	\minipage{0.32\textwidth}
		\centering
		\includegraphics[width=0.7\textwidth, angle=270]{figures/slideseq_cryostat_3.jpg} \hfill
		\caption[Sample 2]{Sample 2}
		\label{fig:slideseq_cryostat_3}
	\endminipage
\end{figure}




\subsubsection{Optimisation of Slide-seq Protocol for Application in Breast Cancer}
The optimised Slide-seq protocol has been published for investigation of murine brain tissue. Breast cancer tissue exhibits distinct histological differences compared to normal brain tissue: marked cellular heterogeneity, variable fibroblast composition, incoherence in spatial architecture. I therefore optimised the Slide-seq protocol for use in breast cancer, with a specific focus on optimising RNA hydribidisation onto the spatial beads. RNA hybridisation involves conflicting factors: sufficient duration to allow for adequate RNA capture whilst maintaining adequate spatial resolution without excessive lateral diffusion of RNA.

In murine brain tissue, RNA hydridisation is performed for 15 minutes. In breast tissue, I explored a range of durations for RNA hybridisation: 15 min, 30 min, 45 min, 60 min, 75 min, 90 min.

Clinical tumour tissue is valuable. The use of clinical tissue must fulfil additional ethical and practical requirements in using patient donated material. Therefore, for protocol optimisation, I used a xenograft sample acquired from Prof. Harris' extensive tissue biobank. Each xenograft sample consists of a FFPE and fresh frozen matched pair. Sample selection was based on high carbonic anhydrase 9 (CA9) expression in FFPE tissue and the downstream Slide-seq protocol was conducted on the matched fresh frozen tissue. FFPE preservation allows for better maintenance of spatial architecture which facilitated sample selection. An optimised protocol for CA9 immunohistochemistry on FFPE tissue was also readily available.

The xenograft sample selected was generated from MDA-231 breast cancer cell line. a triple receptor breast cancer cell line, treated with PBS.
 XXX receptor breast cancer (JLI-164, sample 6, PBS treated)


Xenografts
- histology images
- bioanalyzer: 45, 60, 75, 90 min


Human Samples
- histology
- bioanalyzer
- use of updated version of Slide-seq protocol.


\subsubsection{Modelling of Bead Packing Density in Slide-seq}
slides from confirmation

\subsubsection{Analysis of Slide-seq Profiling in Breast Cancer}
Data required for the completion of this section is pending owing to the exceptional delays and disruption caused by the pandemic. 

\subsubsection{Application and Comparison of Hypoxia Signatures in Slide-seq Findings}
Data required for the completion of this section is pending owing to the exceptional delays and disruption caused by the pandemic. 

\subsubsection{Identification of Tumour and Immune-Specific Hypoxia Markers}
Data required for the completion of this section is pending owing to the exceptional delays and disruption caused by the pandemic. 

\subsection{Application of Single Cell Laser Capture Microdissection in Breast Cancer}
\subsubsection{Tumour and Immune Extraction in Breast Cancer at Single Cell Resolution}
Data required for the completion of this section is pending owing to the exceptional delays and disruption caused by the pandemic. 

\subsubsection{Analysis of Single Cell RNA Sequencing}
Data required for the completion of this section is pending owing to the exceptional delays and disruption caused by the pandemic. 

\subsubsection{Comparison of Slide-seq with Single Cell Laser Capture Microdissection in Serial Sections}
Data required for the completion of this section is pending owing to the exceptional delays and disruption caused by the pandemic. 

\subsection{Application of In Situ Sequencing in Breast Cancer}
\subsubsection{Application of In Situ Sequencing by CARTANA on Breast Cancer}
set up MTA, OCHRe application
hypoxia signature: using published data to identify; those graphs
histology images of 

\subsubsection{Analysis of In Situ Sequencing Data in Breast Cancer}
Data required for the completion of this section is pending owing to the exceptional delays and disruption caused by the pandemic. 

\subsection{Comparison of Expression of Hypoxia Signatures upon Spatial Transcriptomic Profiling in Breast Cancer}
Data required for the completion of this section is pending owing to the exceptional delays and disruption caused by the pandemic. 

\section{Discussion}
I made steps to select most appropriate emerging spatial transcriptomic technology and apply these methods to clinical breast cancer samples. 
Further insights can be gained once the data is available or preliminary analysis has been completed on the available data.


